src/Utils/DEMUtils.js

```diff
@@ -416,7 +283,7 @@ function offsetInExtent(point, extent, target = new THREE.Vector2()) {
     return target.set(originX, originY);
 }
 
-function _readZ(layer, method, coord, nodes, cache) {
+export function _readZ(layer, method, coord, nodes, cache) {
```

Comment:
If it is being exported, you could even merge it with `getTerrainObjectAt`, because it is almost the same return value (`coord` instead of `coord.z`).

Reply:
yes